zap/src/main/java/org/parosproxy/paros/core/scanner/AbstractPlugin.java

```diff
@@ -345,22 +346,11 @@ private void regenerateAntiCsrfToken(HttpMessage msg, AntiCsrfToken antiCsrfToke
         if (tokenValue != null) {
             // Replace token value - only supported in the body right now
             log.debug(
-                    "regenerateAntiCsrfToken replacing "
+                    "regenerateAntiCsrfToken in AbstractPlugin replacing "
```

Comment:
It's not necessary to include the class, the logger is already adding it.

Reply:
I think one of my earlier comments may have lead to this. Sorry